iceoryx_dds/test/moduletests/test_dds_to_iox.cpp

```diff
@@ -90,26 +90,26 @@ TEST_F(DDS2IceoryxGatewayTest, ImmediatelyConnectsConfiguredDataReaders)
     gw.loadConfiguration(config);
 }
 
-TEST_F(DDS2IceoryxGatewayTest, PublishesMemoryChunksContainingSamplesToNetwork)
-{
-    // === Setup
-    auto testService = iox::capro::ServiceDescription({"Radar", "Front-Right", "Reflections"});
+// TEST_F(DDS2IceoryxGatewayTest, PublishesMemoryChunksContainingSamplesToNetwork)
```

Comment:
Just delete these. I will address it in #376.

Reply:
Done. Will be done in #376.